### PR TITLE
OSSM-351 Provide an option to istio-must-gather to scrape only selected SMCP

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Comment by brian-avery** [comment]
This change should go here instead: https://github.com/openshift/openshift-docs/blob/master/modules/ossm-about-collecting-ossm-data.adoc

It will get pulled in automatically when we do a community release.

**Comment by luksa** [comment]
> This change should go here instead: https://github.com/openshift/openshift-docs/blob/master/modules/ossm-about-collecting-ossm-data.adoc
> 
> It will get pulled in automatically when we do a community release.

Oh, so we now source some stuff from the official docs? I thought it was the other way around (but manually).

**Comment by jwendell** [comment]
Yeah, our workflow is weirdly different than vast majority of other projects/teams, even within Red Hat. We do product first, then we backport to community. So, product is our upstream :)

**Comment by brian-avery** [comment]
Product first followed by community is how it was requested of us :) I'm just happy to combine efforts rather than each writing separately.

**Comment by jwendell** [comment]
Yeah, I'm not talking specifically about docs repo.

**Comment by brian-avery** [comment]
Everything in topics and modules is pulled in from product documentation. We have the ability to add patches for Maistra specific content, but ideally all of the docs that engineering and the docs team write are both added directly to the openshift repo and pulled in.

File: modules/ossm-about-collecting-ossm-data.adoc [hunk 1]
```diff
@@ -13,5 +13,27 @@ To collect {ProductName} data with `must-gather`, you must specify the {ProductN
 
 [source,bash]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7
+$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8
```
Comment (jwendell) [inline modules/ossm-about-collecting-ossm-data.adoc:16]: BTW, Is this true? I know that we were using rhel7 for 2.0.x. Have we rebuilt the images on top of rhel8?

Comment (luksa) [reply modules/ossm-about-collecting-ossm-data.adoc:16]: I'll change it back to rhel7. I thought this this was here just because the docs were outdated.

Comment (jwendell) [reply modules/ossm-about-collecting-ossm-data.adoc:16]: If this is for 2.1 then I think it's fine.

**Pull request closed without merging**